tests/utils/test_add_new_model_like.py

```diff
@@ -736,55 +736,6 @@ def test_retrieve_info_for_model_with_bert(self):
         self.assertIsNone(bert_model_patterns.feature_extractor_class)
         self.assertIsNone(bert_model_patterns.processor_class)
 
-    def test_retrieve_info_for_model_pt_tf_with_bert(self):
```

Comment:
It doesn't seem to be a pt-tf test ...?

Reply:
It is in some sense, but we still have the PT bits of this test being tested in `test_retrieve_info_for_model_with_bert` (so this seems to be a leftover copy 👀)